### PR TITLE
RFC: Add keep_all_data_on_mem option

### PR DESCRIPTION
**Pull request by r9y9**
An native implementation (this is what I had locally)

Not deeply investigated yet, but it had seemed to be not working as expected (memory usage wasn't increased when I treid this a while ago). Probably I am not fully understanding how data is processed in espnet. Any ideas?

### CODE REVIEW

File: espnet/tts/pytorch_backend/tts.py [hunk 1]
```diff
@@ -324,6 +325,7 @@ def train(args):
         use_second_target=args.use_second_target,
         preprocess_conf=args.preprocess_conf,
         preprocess_args={'train': False}  # Switch the mode of preprocessing
```
Comment (kan-bayashi) [inline espnet/tts/pytorch_backend/tts.py]:
```suggestion
        preprocess_args={'train': False},  # Switch the mode of preprocessing
```

File: espnet/tts/pytorch_backend/tts.py [hunk 2]
```diff
@@ -316,6 +316,7 @@ def train(args):
         use_second_target=args.use_second_target,
         preprocess_conf=args.preprocess_conf,
         preprocess_args={'train': True}  # Switch the mode of preprocessing
```
Comment (kan-bayashi) [inline espnet/tts/pytorch_backend/tts.py]:
```suggestion
        preprocess_args={'train': True},  # Switch the mode of preprocessing
```

Comment (r9y9) [reply espnet/tts/pytorch_backend/tts.py]: Ah, this is due to cherry-picking conflict. Will push a fix shortly.

**Comment by kan-bayashi** [comment]
LGTM
@kamo-naoyuki Do you have any comments?

**Comment by kamo-naoyuki** [comment]
LGTM. 
(Perhaps I also modify this feature with #1012, but it's O.K in the current status.)

**Comment by r9y9** [comment]
Yeah maybe we can close this PR. I would be grateful if you could consider the similar feature in #1012. cc: @kamo-naoyuki

**Pull request merged**